src/meinberlin_lib/meinberlin_lib/static/js/Packages/Meinberlin/MeinberlinDe/Module.ts

```diff
@@ -51,13 +54,13 @@ export var register = (angular) => {
                 RIAlexanderplatzProcess.content_type, "mein.berlin.de")(adhResourceAreaProvider);
 
             var registerRoutes1 = AdhIdeaCollectionWorkbench.registerRoutesFactory(
-                RIBuergerhaushaltProcess, RIBuergerhaushaltProposal, RIBuergerhaushaltProposalVersion);
+                RIBuergerhaushaltProcess, RIBuergerhaushaltProposal, RIBuergerhaushaltProposalVersion, true);
             registerRoutes1("mein.berlin.de")(adhResourceAreaProvider);
             var registerRoutes2 = AdhIdeaCollectionWorkbench.registerRoutesFactory(
-                RIIdeaCollectionProcess, RIGeoProposal, RIGeoProposalVersion);
+                RIIdeaCollectionProcess, RIGeoProposal, RIGeoProposalVersion, true);
             registerRoutes2("mein.berlin.de")(adhResourceAreaProvider);
             var registerRoutes3 = AdhIdeaCollectionWorkbench.registerRoutesFactory(
-                RIKiezkasseProcess, RIKiezkasseProposal, RIKiezkasseProposalVersion);
+                RIKiezkasseProcess, RIKiezkasseProposal, RIKiezkasseProposalVersion, true);
             registerRoutes3("mein.berlin.de")(adhResourceAreaProvider);
```

Comment:
Due to rebaseing, `registerRoutes4` (see b82df69) somehow got lost. I fixed it in 2303ecc.